src/lib/pay-request/api_utils/connector.js

```diff
@@ -104,6 +104,21 @@ const connectorMethods = function connectorMethods(instance) {
     })
   }
 
+  const updateStripeSetupValues = async function updateStripeSetupValues(id, stripeSetupFields) {
+    const url = `/v1/api/accounts/${id}/stripe-setup`
+    const payload = []
+    
+    stripeSetupFields.forEach((field) =>{
+      payload.push({
+        op: 'replace',
+        path: field,
+        value: true
+      })
+    })
+
+    await axiosInstance.patch(url, payload)
```

Comment:
If this returns the promise rather than waiting for it the error propagation up through the calling code should be easier to understand
```suggestion
    return axiosInstance.patch(url, payload)
```

Given the above, this method just returns the aysnc call further up so the function would no longer need to be `async`

Reply:
Good idea - just updated it.
Thanks